test: addition of last tests functions

lorenzodidomenic opened the pull request.

lorenzodidomenic committed:
test: addition of last tests functions

AllegraSalvatore committed:
Update test_functions.py

Modifico:
tests/test_functions.py:231:0: C0303: Trailing whitespace (trailing-whitespace)
tests/test_functions.py:250:0: C0303: Trailing whitespace (trailing-whitespace)
tests/test_functions.py:269:0: C0303: Trailing whitespace (trailing-whitespace)

lorenzodidomenic committed:
fix: fix of some pylint error

lorenzodidomenic committed:
Merge branch 'last_last_branch' of github.com:lorenzodidomenic/Botquake into last_last_branch

lorenzodidomenic committed:
fix: fix of some pylint error

lorenzodidomenic committed:
fix: fix of some pylint error

lorenzodidomenic committed:
fix: fix of some pylint error

AllegraSalvatore reviewed (commented):
Un altro tassello verso il 100% di coverage. @lorenzodidomenic

Commit:
Modifica la logica della gestione dei messaggi
Riprende l'intento iniziale della pull request #30
modifica il tet e ne incrementa la coverage

linuchs commented:
L'ultima commit riprende il codice di @lorenzodidomenic all'inizio di questa Pull Request permettendoci di passare dall'86% all'88% di coverage.
